### PR TITLE
Add bangla localization

### PR DESCRIPTION
**Pull request by asem-hamid**
## Checklist

- [x] I read and understood the **[Contributing Guidelines](https://github.com/Abrynos/ShoppingList/blob/main/.github/CONTRIBUTING.md)**.
- [x] This is not a **[duplicate](https://github.com/Abrynos/ShoppingList/pulls)** of an existing merge request.
- [ ] This is not a **[translation update](https://crowdin.com/project/abrynosshoppinglist)**.
- [x] I believe this falls into the scope of the project and should be part of the built-in functionality.
- [x] My code follows the **[code style](https://github.com/Abrynos/ShoppingList/blob/main/.github/CONTRIBUTING.md#code-style)** of this project.
- [x] All new and existing tests pass.

### New functionality

Add bangla localization

Fix : #131

### CODE REVIEW

File: fastlane/metadata/android/en-US/full_description.txt [hunk 1]
```diff
@@ -10,6 +10,7 @@ This is the a simple Android application serving the purpose of a shopping list.
 
 <b>Languages:</b>
 * Azerbaijani
+* Bangla
```
Comment (Abrynos) [inline fastlane/metadata/android/en-US/full_description.txt:13]: This list uses the international names of the languages. As such, this should say `Bengali`

Comment (Abrynos) [reply fastlane/metadata/android/en-US/full_description.txt:13]: I like that you already put it in the correct alphabetical order tho.

File: app/src/main/res/values-bn/strings.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License along
+  ~ with this program; if not, write to the Free Software Foundation, Inc.,
+  ~ 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+  -->
+
+<resources>
```
Comment (Abrynos) [inline app/src/main/res/values-bn/strings.xml:18]: Some strings should not be translated as indicated by the `translatable` attribute being set to `false`. Crowdin already takes care of that.

Remove the corresponding entries from the file.

**Review by Abrynos — CHANGES_REQUESTED** [review]
Also missing entry in `/fastlane/metadata/android/en-US/changelogs/19.txt` (before the already present default-entry):
```md
* Added translations for Bengali
```

File: app/src/main/res/values-bn/strings.xml (hunk 2 above)
Comment (asem-hamid) [reply app/src/main/res/values-bn/strings.xml:18]: This 5 ?

image

File: fastlane/metadata/android/en-US/full_description.txt (hunk 1 above)
Comment (asem-hamid) [reply fastlane/metadata/android/en-US/full_description.txt]: done

File: app/src/main/res/values-bn/strings.xml (hunk 2 above)
Comment (asem-hamid) [reply app/src/main/res/values-bn/strings.xml:18]: Done

**Review by Abrynos — CHANGES_REQUESTED** [review]

File: fastlane/metadata/android/en-US/changelogs/20.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+* Added translations for Bengali
```
[the remark this answers is not in the document]
Comment (asem-hamid) [reply fastlane/metadata/android/en-US/changelogs/20.txt:1]: oh , my bad .

Should add after that or remove the old one
image

[the remark this answers is not in the document]
Comment (Abrynos) [reply fastlane/metadata/android/en-US/changelogs/20.txt:1]: > Should add after that or remove the old one

Before it. Same way it's done with all the other changelogs 😉

[the remark this answers is not in the document]
Comment (Abrynos) [reply fastlane/metadata/android/en-US/changelogs/20.txt]: Reason for this whole "needs to match"-thing is, that this file is automatically used for both the text of the GitHub-release and the one on F-Droid.

[the remark this answers is not in the document]
Comment (asem-hamid) [reply fastlane/metadata/android/en-US/changelogs/20.txt]: Done

**Comment by Abrynos** [comment]
LGTM. Need to fix the pipeline for merge requests tho...

**Review by Abrynos — APPROVED** [review]

**Comment by Abrynos** [comment]
@asem-hamid  Should work now. Can you merge with the main branch again?

**Comment by asem-hamid** [comment]
Did you mean resubmit a PR ??  Only Maintainer can merge a PR .

**Comment by Abrynos** [comment]
> Did you mean resubmit a PR

I meant what I just did to your branch 😉

**Pull request merged**